clients/ui/frontend/src/__tests__/cypress/cypress.config.ts

```diff
@@ -0,0 +1,122 @@
+import path from 'path';
+import fs from 'fs';
+import { defineConfig } from 'cypress';
+import coverage from '@cypress/code-coverage/task';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore no types available
+import cypressHighResolution from 'cypress-high-resolution';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore no types available
+import { beforeRunHook, afterRunHook } from 'cypress-mochawesome-reporter/lib';
+import { mergeFiles } from 'junit-report-merger';
+import { env, BASE_URL } from '~/src/__tests__/cypress/cypress/utils/testConfig';
+
+
+const resultsDir = `${env.CY_RESULTS_DIR || 'results'}/${env.CY_MOCK ? 'mocked' : 'e2e'}`;
+
+export default defineConfig({
+  experimentalMemoryManagement: true,
+  // Use relative path as a workaround to https://github.com/cypress-io/cypress/issues/6406
+  reporter: '../../../node_modules/cypress-multi-reporters',
+  reporterOptions: {
+    reporterEnabled: 'cypress-mochawesome-reporter, mocha-junit-reporter',
+    mochaJunitReporterReporterOptions: {
+      mochaFile: `${resultsDir}/junit/junit-[hash].xml`,
+    },
+    cypressMochawesomeReporterReporterOptions: {
+      charts: true,
+      embeddedScreenshots: false,
+      ignoreVideos: false,
+      inlineAssets: true,
+      reportDir: resultsDir,
+      videoOnFailOnly: true,
+    },
+  },
+  chromeWebSecurity: false,
+  viewportWidth: 1920,
+  viewportHeight: 1080,
+  numTestsKeptInMemory: 1,
+  video: true,
+  screenshotsFolder: `${resultsDir}/screenshots`,
+  videosFolder: `${resultsDir}/videos`,
+  env: {
+    MOCK: !!env.CY_MOCK,
+    coverage: !!env.CY_COVERAGE,
```

Comment:
Same as this, we got a `cypress:run:mock_coverage` command with that env variable set to true, but here it will be always false, should we add that command or just handle this?